### ISSUE TITLE
Music stops mid-playback

### ISSUE THREAD

**Issue by CringyAdam** [nexu-dev/discord.js-music#10]
The music stops in the middle of the song.
I need help asap as I use this addon in a server of 1000 users.

**Comment by godleydemon** [comment]
having same bug on windows 7 64bit running discord.js. Also having a strange bug where I can't change the prefix from anything but "-" o.0

**Comment by SpaceX9456** [comment]
Please, when will this be fixed

**Comment by ElizaDev** [comment]
I have the same issue. Windows 10 64 bit never completes a full song, but begins playing. On Raspberry Pi 2, it states its playing the song and immediately cuts off and ends the song.

**Comment by camer0** [comment]
I seem to have fixed it by removing the filter completely

**Comment by yorusi** [comment]
same issue here and i cant solve the problem with this code
`let dispatcher = connection.playStream(ytdl(video.webpage_url), {highWaterMark: 163840, retries: 100}, {seek: 1, volume: (DEFAULT_VOLUME/100)});`

**Comment by camer0** [comment]
Yorusi, you are right. It seemed to only temporarily fix the problem for me

**Comment by yorusi** [comment]
camer0, thank you so much. i solved the problem with your help
it works well

**Comment by camer0** [comment]
No problem, glad it works!

**Comment by MFNich** [comment]
I'm getting a  throw er; // Unhandled 'error' event TypeError: Invalid non-string/buffer chunk. When replacing 
let dispatcher = connection.playStream(stream(video.webpage_url))
 with 
let dispatcher = connection.playStream(ytdl(video.webpage_url, {filter: 'audioonly'}), {seek: 0, volume: (DEFAULT_VOLUME/100)});

anyone know why?

**Comment by Philip-j-fry** [comment]
@MFNich 
Did you make sure to install youtube-audio-stream and add const stream = require('youtube-audio-stream'); I had this issue and this fixed it for me.

**Comment by MFNich** [comment]
@Philip-j-fry 
Yeah I installed and assigned  const stream = require('youtube-audio-stream');

**Comment by relaytheory** [comment]
let dispatcher = connection.playStream(stream(video.webpage_url))
ReferenceError: connection is not defined

Am I missing a dependency?

**Comment by relaytheory** [comment]
Same error as MFNich.
Bot joins the channel fine, no audio plays.

**Comment by Philip-j-fry** [comment]
Need to set dispatcher too let dispatcher = connection.playStream(stream(video.webpage_url))
This fixed the issue for me

**Comment by IWasAndrerstajn** [comment]
Where do i add let dispatcher = connection.playStream(stream(video.webpage_url))
